.gitignore

```diff
@@ -139,3 +139,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.DS_Store
+
+run-dlp/LocalTestingFunctions/*
+run-dlp/supporting-documents/*
```

Comment:
Not crucial but I wonder if the folder name can be more descriptive? One idea: storage-dlp?